pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.cpp

```diff
@@ -134,13 +134,20 @@ TrtUniquePtr<nvinfer1::ICudaEngine> TensorrtOnnxParser::BuildEngine(
     }
     auto trt_engine = TrtUniquePtr<nvinfer1::ICudaEngine>(
         context.builder->buildEngineWithConfig(*context.network, *config));
-    if (trt_engine == nullptr) {
-      LOG(INFO) << "Failed to build the engine, error message are:";
+
+    // We had meet segfault when retrive error message after build engine
+    // failed. It's a issue of TensorRT.
+    // To make our process more robust we would like to enable log
```

Comment:
When `TORCH_BLADE_DEBUG_LOG` is set, the following code will still try to get error msg and run into the same segfault?

Reply:
Yes, but occasionally. We had only run into the segfault when trying to optimize TransformerXL.